CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Added
+
+### Fixed
+- Replaced deprecated 'form' type by FormType class
+
+
```

Comment:
Please don't touch the CHANGELOG, and indicate your changes in the PR body instead, this should spare you from some conflicts.